x-pack/test/reporting/functional/reporting.js

```diff
@@ -135,7 +135,8 @@ export default function({ getService, getPageObjects }) {
         it('matches baseline report', async function() {
           this.timeout(300000);
 
-          await PageObjects.dashboard.switchToEditMode();
+          await PageObjects.common.navigateToApp('dashboard');
+          await PageObjects.dashboard.gotoDashboardEditMode('My PNG Dash');
```

Comment:
Thanks for catching this, and sorry you had to go through it. There definitely is an issue with the Reporting functional tests in that they have overcomplicated steps that could be worked around by using smarter archives of the objects being reported on.